content/concepts/plugins.md

```diff
@@ -106,6 +106,46 @@ Disabling a plugin is simple. Either remove the plugin files in the plugins fold
 
 Alternatively, it is also valid to disable a plugin by renaming a plugin file from `plugin-before.conf` to `plugin-before.conf.disabled`.
 
+## Conditionally enable plugins for multi-application environments
+
+If CRS is installed on a reverse-proxy or a web server with multiple web applications, then you may wish to only enable certain plugins (such as rule exclusion plugins) for certain `VirtualHosts`. This ensures that rules designed for a specific web application are only enabled for the intended web application, reducing the scope of any possible bypasses within a plugin.
```

Comment:
```suggestion
If CRS is installed on a reverse-proxy or a web server with multiple web applications, then you may wish to only enable certain plugins (such as rule exclusion plugins) for certain virtual hosts (`VirtualHost` Apache httpd, `location` in nginx). This ensures that rules designed for a specific web application are only enabled for the intended web application, reducing the scope of any possible bypasses within a plugin.
```

Reply:
[Location](https://nginx.org/en/docs/http/ngx_http_core_module.html#location) directive in Nginx is equivalent to [Apache's Directory](https://httpd.apache.org/docs/2.4/mod/core.html#directory) directive, both respectively live within the Server Block and Virtual Host context.

Reply:
Right. So `Server` and `VirtualHost` then.